example/lib/main.dart

```diff
@@ -48,9 +49,9 @@ class MyHomePage extends StatelessWidget {
       print('Joining room: name=$name, roomName=$roomName');
     }
     try {
-      final details = await fetchConnectionDetails(name, roomName);
-      await roomCtx.connect(
-          url: details.serverUrl, token: details.participantToken);
+      //final details = await fetchConnectionDetails(name, roomName);
+      //await roomCtx.connect(
+      //    url: details.serverUrl, token: details.participantToken);
```

Comment:
was this intentionally removed?

Reply:
This is the code for fetching token from meet.staging.livekit.io. Remove it for now and consider adding it when building the example web app later.